src/char/char.hpp

```diff
@@ -14,19 +14,30 @@
 
 #include "packets.hpp"
 
+using namespace rathena::server_core;
```

Comment:
we shouldn't be doing this in a header file, we're polluting the global namespace of every file that includes `char.hpp`

instead, we can specifically import Core, with
`using rathena::server_core::Core`